bigtop-packages/src/common/logstash/do-component-build

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+. `dirname $0`/bigtop.bom
+
+export RELEASE=1
+export LANG="en_US.UTF-8"
+
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+fi
+
+case ${ID} in
+    ubuntu | debian)
+        sudo apt-get -y update
+        sudo apt-get install -y gnupg2
+        ;;
+esac
+
+case ${ID}-${VERSION_ID} in
+    ubuntu-18* | debian-9)
+    gpg2 --list-keys
+    echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
```

Comment:
Would you mind to share what's the purpose of this line?

Reply:
There are the IPv6 issues happening in docker containers on ubuntu18 and debian-9:
```
+ gpg2 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
gpg: keyserver receive failed: Cannot assign requested address
```
The soution was provided by rvm.io: https://rvm.io/rvm/security#ipv6-issues